app/javascript/controllers/reviewer_sunets_controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller  {
+  static targets = ["container", "error"];
```

Comment:
i'm not seeing the container target defined in the html here...is it part of an already existing component?